topic/topiclistener/event_handler.go

```diff
@@ -76,10 +76,8 @@ func (b BaseHandler) OnReadMessages(
 }
 
 type (
-	StartPartitionSessionRequest = topiclistenerinternal.PublicStartPartitionSessionEvent
-	StartPartitionSessionConfirm = topiclistenerinternal.PublicStartPartitionSessionConfirm
-	StopPartitionSessionRequest  = topiclistenerinternal.PublicStopPartitionSessionEvent
-	StopPartitionSessionConfirm  = topiclistenerinternal.PublicStopPartitionSessionConfirm
+	StartPartitionSessionEvent = topiclistenerinternal.PublicStartPartitionSessionEvent
```

Comment:
Maybe events will be named as enum constants such as
- EventStartPartitionSession
- EventStopPartitionSession
etc 
?